appconfig/src/main/kotlin/io/element/android/appconfig/AuthenticationConfig.kt

```diff
@@ -21,4 +21,6 @@ object AuthenticationConfig {
 
     const val DEFAULT_HOMESERVER_URL = MATRIX_ORG_URL
     const val SLIDING_SYNC_READ_MORE_URL = "https://github.com/matrix-org/sliding-sync/blob/main/docs/Landing.md"
+
+    val SLIDING_SYNC_PROXY_URL: String? = null
```

Comment:
Maybe document this `val`? Something like "Force a sliding sync proxy url, if not null, the proxy url in the .well-known file will be ignored"

Reply:
https://github.com/element-hq/element-x-android/pull/3068/commits/ab237eb89774f38a2a836eee89c4c3821056f830